tools/scylla-nodetool.cc

```diff
@@ -520,15 +520,9 @@ void help_operation(const tool_app_template::config& cfg, const bpo::variables_m
         // goes in.
 
         bpo::options_description opts_desc(fmt::format("scylla-{} options", app_name));
```

Comment:
> tools: add constant 

autocorrect? don't see what the constant is.

Ah extracting standard command line options to a constant.